### PR TITLE
Add dataset_class argument to from_pandas

### PR DESCRIPTION
**Pull request by jtilly**
I added the `dataset_class` argument to `from_pandas` to make it easier to use custom expectations.

### CODE REVIEW

File: great_expectations/util.py [hunk 1]
```diff
@@ -123,10 +123,27 @@ def read_parquet(
     return df
 
 
-def from_pandas(pandas_df, expectations_config=None, autoinspect_func=None):
+def from_pandas(pandas_df,
```
Comment (abegong) [inline great_expectations/util.py:126]: This docstring is a thing of beauty!

**Review by abegong — CHANGES_REQUESTED** [review]
This is great!

One request: can you add a test or two to exercise the new `dataset_class` parameter?

**Review by abegong — APPROVED** [review]
Thanks for adding tests. Merging now.

**Comment by jtilly** [comment]
Awesome, thanks for including this.

**Pull request merged**